metadata-ingestion/examples/library/dataset_schema.py

```diff
@@ -1,65 +1,20 @@
-# Imports for urn construction utility methods
-from datahub.emitter.mce_builder import make_data_platform_urn, make_dataset_urn
-from datahub.emitter.mcp import MetadataChangeProposalWrapper
-from datahub.emitter.rest_emitter import DatahubRestEmitter
+from datahub.sdk import DataHubClient, Dataset
 
-# Imports for metadata model classes
-from datahub.metadata.schema_classes import (
-    AuditStampClass,
-    DateTypeClass,
-    OtherSchemaClass,
-    SchemaFieldClass,
-    SchemaFieldDataTypeClass,
-    SchemaMetadataClass,
-    StringTypeClass,
-)
+client = DataHubClient.from_env()
 
-event: MetadataChangeProposalWrapper = MetadataChangeProposalWrapper(
-    entityUrn=make_dataset_urn(platform="hive", name="realestate_db.sales", env="PROD"),
-    aspect=SchemaMetadataClass(
-        schemaName="customer",  # not used
-        platform=make_data_platform_urn("hive"),  # important <- platform must be an urn
-        version=0,  # when the source system has a notion of versioning of schemas, insert this in, otherwise leave as 0
-        hash="",  # when the source system has a notion of unique schemas identified via hash, include a hash, else leave it as empty string
-        platformSchema=OtherSchemaClass(rawSchema="__insert raw schema here__"),
-        lastModified=AuditStampClass(
-            time=1640692800000, actor="urn:li:corpuser:ingestion"
+dataset = Dataset(
+    platform="hive",
+    name="realestate_db.sales",
+    schema=[
+        # tuples of (field name, data type, description)
```

Comment:
we may keep here the legacy names for the old-school users 

```suggestion
        # tuples of (field path, native data type, description)
```

also, what about the `type`, is it inferred from the native data type?